modules/react/form-field/lib/FormField.tsx

```diff
@@ -231,7 +232,13 @@ export class FormField extends React.Component<React.PropsWithChildren<FormField
       );
     } else {
       return (
-        <FormFieldContainer useFieldset={useFieldset} labelPosition={labelPosition} {...elemProps}>
+        <FormFieldContainer
+          id="foo"
+          useFieldset={useFieldset}
+          labelPosition={labelPosition}
+          grow={grow}
+          {...elemProps}
```

Comment:
idk why this prop wasn't forwarded here. If usefieldset is not used, consumers should still be able to set width to 100%. Look at `grow` story for select for reference.